api/queries_repo.go

```diff
@@ -312,12 +313,23 @@ type repositoryV3 struct {
 }
 
 // ForkRepo forks the repository on GitHub and returns the new repository
-func ForkRepo(client *Client, repo ghrepo.Interface) (*Repository, error) {
+func ForkRepo(client *Client, repo ghrepo.Interface, org string) (*Repository, error) {
 	path := fmt.Sprintf("repos/%s/forks", ghrepo.FullName(repo))
 	body := bytes.NewBufferString(`{}`)
+	if org != "" {
+		orgBody := fmt.Sprintf(`{ "organization": "%s" }`, org)
+		body = bytes.NewBufferString(orgBody)
+	}
+
 	result := repositoryV3{}
 	err := client.REST(repo.RepoHost(), "POST", path, body, &result)
 	if err != nil {
+		var httpErr HTTPError
+		if errors.As(err, &httpErr) {
+			if httpErr.StatusCode == http.StatusForbidden {
+				return nil, fmt.Errorf("Must have admin rights to Repository")
```

Comment:
I appreciate you adding this special case, but I think it might be premature optimization. The 401 error will be reported to the terminal via our usual error handling and I don't think we need to add anything on top of that. If the user doesn't have write access to their org, the fork command will fail with a visible 401 and it will be enough information for them to go on.